sync/config/pipelines.yaml

```diff
@@ -135,10 +135,10 @@ tags:
       - deprecations.md
       - install.md
       - tutorial.md
-- name: master
+- name: main
   # The name to display on tekton.dev.
   # sync.py will use this value in the version switcher and other places.
-  displayName: master
+  displayName: main
```

Comment:
These are already in, they shouldn't be part of this PR

Reply:
thank you for your review. to reslove the deploy failure I did this merge to our PR. but it seems not right. I will resolve this with my fellow.

Reply:
Hi @afrittoli , To remove these excess content, I create another PR #263 with  all modifications in landing page Only. Could you please give a review? I will close this PR today.